content/en/events/kcc2021/faq.md

```diff
@@ -0,0 +1,204 @@
+[activities] page.
+
+[activities]: /events/kcc2021/activities/
+
+
+## Discord Information
+
+### How do I join?
+
+Please see the [How to join] page.
+
+[How to join]: /events/kcc2021/how-to-join/
+
+### Why do I need to verify myself once I connect (complete captcha)?
+
+Spam bots and raids are more common with Discord than other platforms such as
+slack. Completing the external Captcha helps us keep Discord a safe place to
+have fun <3
+
+
+### Why do I need to agree to abide by the Code of Conduct?
+
+Much like other communication platforms used by Kubernetes (and other CNCF
+projects) we want this to be an inclusive and inviting place for everyone. We
+understand that certain things like profane language are used by many while
+gaming, please still try to be inclusive and not offensive. 
+
+Remember: Don’t be a jerk. :) 
+
+
+### Why can I only see the instructions-to-connect and info channels?
+
+You’ve completed the captcha, but haven’t agreed to the code of conduct, head
+over to the `#code-of-conduct` channel and accept it, and you’ll be able to see
+everything. If you need assistance, join the `#help` channel.
+
+
+### What is this YAGPDB.xyz and why is it messaging me?
+
+<img align="left" src="/events/kcc2021/yagpdb.png" width="40" height="40">
+
+[YAGPDB] (Yet Another General Purpose Discord Bot) is a verified discord bot
+that we use to manage a variety of services within discord, from verification
+and moderation to role selection and automation.
+
+
+### How do I display my pronouns in Discord?
+
+In Discord, you can set a nickname per-”server”. We recommend putting your
+pronouns in your nickname so they are visible at all times in Discord and with
+overlays. A few examples of what this would look like:
+- `Arty (they/them)`
+- `Bob (he/him)`
+- `Jamie (she/her)`
+
+1 ) To do so, click the server dropdown (Top Left)
+
+<img align="center" src="/events/kcc2021/server.png" width="40%">
+<br>
+<br>
+2 ) Select “Change Nickname”
+<br>
+<br>
+<img align="center" src="/events/kcc2021/nickname.png" width="40%">
+<br>
+<br>
+3 ) Then follow the template of “nickname (pro/noun)” like so:
+<br>
+<br>
+<img align="center" src="/events/kcc2021/name.png" width="40%">
+
+
+### What are the `#<channel>-info` channels for?
+
+For games that are popular enough to have their own category, the info channel
+in them will describe the game, links to where it can be downloaded, and will
+provide guidance on reporting people for that game/platform.
+
+### What are the `#<channel>-links` channels for?
+
+Often when you’re having a conversation in one of the voice channels you’ll want
+to share a link with the other people on voice. You can’t do this directly in
+the voice channels, and it can clutter up the main text-chat channels, so the
+`#<channel>-links` channels are a scratch-pad for links or other things that
+are only relevant to the conversation carrying on in voice chat.
```

Comment:
We're removing these this time around (will also need to be removed from the ToC)